src/utilities/io.jl

```diff
@@ -45,7 +45,7 @@ function getjuliatype(s::Sample, v::Symbol, cached_syms=nothing)
     # Fill sample
     for i = 1:length(syms)
         # Get indexing
-        idx = Main.eval(parse(idx_comp[i][1]))
+        idx = Main.eval(Meta.parse(idx_comp[i][1]))
```

Comment:
`parse(Int, idx_comp[i][1])` seems to be what's needed here. There is also another `eval(parse(..))` above which should be replaced the same way.

Reply:
Doesn't the `Main.eval(Meta.parse(. . . ))` functionality allow for non-integer indices? Don't we have cases where we index by nonintegers?

Reply:
You are correct. But note that because of the way `idx` is used later, it can only be a tuple, array or integer. The following is an `eval`-free way of doing the right thing:
```julia
function parse_inds(inds)
	p_inds = [parse(Int, m.captures[1]) for m in eachmatch(r"(\d+)", inds)]
	if length(p_inds) == 1
		return p_inds[1]
	else
		return Tuple(p_inds)
	end
end
parse_inds("(1,1)")
parse_inds("1")
```